app/views/comments/_comment_form.html.erb

```diff
@@ -5,7 +5,7 @@
 <div class="post comment" id="comment_form_for_<%= commentable.id %>">
   <%= form_for value_for_comment_form(commentable, comment), remote: !comment.new_record?, authenticity_token: true, html: { id: "comment_for_#{commentable.id}" } do |f| %>
     <fieldset>
-      <legend><%= ts("Post Comment") %></legend>
+      <legend><%= t(".form_title") %></legend>
```

Comment:
Based on other places where there is a `<legend>`, maybe the key should be `.legend`. Once again this is meta information that is not visible usually, so it's nice to make that clear.